rules/falco_rules.yaml

```diff
@@ -1936,17 +1941,84 @@
     WARNING
   tags: [process, mitre_credential_access]
 
+- list: log_directories
+  items: [/var/log, /dev/log]
+
+- list: log_files
+  items: [syslog, auth.log, secure, kern.log, cron, user.log, dpkg.log, last.log, yum.log, access_log, mysql.log, mysqld.log]
+
+- macro: access_log_files
+  condition: (fd.directory pmatch (log_directories) or fd.filename in (log_files))
```

Comment:
If you're trying to match a specific directory against a set of other directories, you can just use `fd.directory in (log_directories)`. You only need to use pmatch when you're trying to match a full path against a set of directories.